script/VaultsDeployer.s.sol

```diff
@@ -36,6 +36,10 @@ contract VaultsActionBatcher is SpokeActionBatcher {
         report.syncDepositVaultFactory.rely(address(report.spoke.spoke));
         report.asyncRequestManager.rely(address(report.spoke.spoke));
 
+        // Rely VaultRegistry
+        report.asyncVaultFactory.rely(address(report.spoke.vaultRegistry));
```

Comment:
Above these lines we have a ward from the vault factories to the spoke which can be removed AFAICT

```diff
-        report.asyncVaultFactory.rely(address(report.spoke.spoke));
-        report.syncDepositVaultFactory.rely(address(report.spoke.spoke));
```